app/src/components/Header.tsx

```diff
@@ -20,7 +17,7 @@ const primaryLinks: {
     i18nKey: "nav_link_health",
     href: "/health",
   },
-];
+] as const;
```

Comment:
ℹ️ You can't pass in a `string` type to `t()`. This makes it so `i18nKey` is `"nav_link_home" | "nav_link_health"`